types/google.visualization/index.d.ts

```diff
@@ -261,7 +279,7 @@ declare namespace google {
             getTableColumnIndex(viewColumnIndex: number): number;
             getTableRowIndex(viewRowIndex: number): number;
             getViewColumnIndex(tableColumnIndex: number): number;
-            getViewColumns(): number[];
+            getViewColumns(): any[];
```

Comment:
This is actually incorrect in the documentation. If you create a derived/calculated column in the DataView, it will return it, which has an object structure. Since the union of number[] or a new object doesn't improve typechecking, this seemed the appropriate correction.

Reply:
Seems better to specify types, IDE will show developers notes.
Could you please add types that specified in documentation?
https://developers.google.com/chart/interactive/docs/reference#DataView
`columnIndexes - An array of numbers and/or objects (can be mixed)` section

Reply:
By providing both possibilities, it made sense to switch any to ColumnSpec